### DISCUSSION TITLE
[Enhancement] [select] Add popperOptions prop

### DISCUSSION THREAD

**Discussion by hahatulk** [element-plus/element-plus#11631] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select`

## Description

Please add prop popperOptions to control popperjs

### Additional comments

Also would be great to add it on every dropdown component in element-plus

**Comment by ryuhangyeong** [comment]
https://github.com/element-plus/element-plus/pull/11734

> Also would be great to add it on every dropdown component in element-plus

I'll investigate other components as well and add them if I can. thank you!
